services/app-web/src/components/HealthPlanPackageTable/HealthPlanPackageTable.test.tsx

```diff
@@ -93,6 +93,9 @@ const submissions: PackageInDashboardType[] = [
 ]
 
 describe('HealthPlanPackageTable for CMS User (with filters)', () => {
+    beforeEach(() => {
+        window.location.assign('#')
```

Comment:
is this to clear any state leftover between tests?

Reply:
yup